client/src/common/ui/loadingLinear/index.js

```diff
@@ -1,12 +1,39 @@
-import { LinearProgress } from '@mui/material'
+import { Box, LinearProgress } from '@mui/material'
+import { useRouter } from 'next/router'
+import { useEffect } from 'react'
+import { updateSyncV, useSyncV } from 'use-sync-v'
 
 export const LoadingLinear = () => {
+  const authLoading = useSyncV('auth.loading')
```

Comment:
Currently, we do not have an `auth.loading` use-sync-v store field value. Using it like this always assigns an `undefined` value to the `authLoading` variable.

Kindly see the use-sync-v `auth` store fields definition in the `/client/src/lib/store/index.js`. If you meant to listen for the Firebase Auth "loading" status, pls use `auth.authLoading`.

#### P.S. You can rename

`auth.authLoading` => `auth.loading` if this sounds more consistent, but please be sure to update all `auth.authLoading` references in the codes.